repobee_sanitizer/_sanitize_repo.py

```diff
@@ -42,10 +48,24 @@ def _sanitize_repo(
         if args.no_commit:
             LOGGER.info("Executing dry run")
             file_relpaths = _discover_dirty_files(args.repo_root)
-            _sanitize_files(args.repo_root, file_relpaths)
+            errors = _sanitize_files(args.repo_root, file_relpaths)
+            if errors:
+                return plug.Result(
+                    name="sanitize-repo",
+                    msg=_format_error_string(errors),
+                    status=plug.Status.ERROR,
+                )
```

Comment:
This is duplicated here and in the else branch. Put it after the entire if-statement instead.
```suggestion
```